analyses/fusion_filtering/results/fusion-recurrently-fused-genes-bycancergroup.tsv

```diff
@@ -0,0 +1,10 @@
+Gene	cancer_group	count
+BRAF	Low-grade glioma/astrocytoma	121
+KIAA1549	Low-grade glioma/astrocytoma	114
+C11orf95	Ependymoma	28
+RELA	Ependymoma	25
+EWSR1	Ewing sarcoma	6
+FLI1	Ewing sarcoma	6
+NTRK2	Low-grade glioma/astrocytoma	5
+RAF1	Low-grade glioma/astrocytoma	4
+ROS1	High-grade glioma/astrocytoma	4
```

Comment:
- MET is unexpected deletion here.
According to the sample selection done in `dev` in the `06` code the distribution is as:
```
.
Diffuse intrinsic pontine glioma           Diffuse midline glioma    High-grade glioma/astrocytoma 
                               1                                5                                2 
```
BS_IDs used as independent sample list in `dev`:
```
# A tibble: 8 x 2
  Kids_First_Biospecimen_ID cancer_group                    
  <chr>                     <chr>                           
1 BS_45J79YTF               High-grade glioma/astrocytoma   
2 BS_4QW2S1Q8               Diffuse midline glioma          
3 BS_6DCSD5Y6               Diffuse intrinsic pontine glioma
4 BS_7327SDAQ               High-grade glioma/astrocytoma   
5 BS_8ZY4GST0               Diffuse midline glioma          
6 BS_97M1E2DW               Diffuse midline glioma          
7 BS_GWSJ4Z9H               Diffuse midline glioma          
8 BS_NGSG2KB6               Diffuse midline glioma 
```

In the current code the sample selection seems to be:
```
# A tibble: 5 x 2
  Kids_First_Biospecimen_ID cancer_group                    
  <chr>                     <chr>                           
1 BS_45J79YTF               High-grade glioma/astrocytoma   
2 BS_4QW2S1Q8               Diffuse midline glioma          
3 BS_6DCSD5Y6               Diffuse intrinsic pontine glioma
4 BS_7327SDAQ               High-grade glioma/astrocytoma   
5 BS_97M1E2DW               Diffuse midline glioma     
```






- SCFD2 is expected change, removed because of cancer_group update
```
Diffuse intrinsic pontine glioma           Diffuse midline glioma 
                               1                                3 


```

Reply:
I see the missing RNA-Seq samples (`BS_NGSG2KB6`, `BS_GWSJ4Z9H` and `BS_8ZY4GST0`)  in the update rna-seq independent sample file so I believe my suggested update in  https://github.com/PediatricOpenTargets/OpenPedCan-analysis/pull/41#discussion_r666212786 might help get back the MET recurrently fused status ( not tested in code)

Reply:
Thanks so much for looking into this! I will change the code for getting RNA samples + add cohort as parameter + re-run.